src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC, SVGProps, useEffect, useRef } from 'react'
+import { icons } from '@/assets/icons'
+
+interface IconProps {
+  size: 'sm' | 'md' | 'lg';
+  name: string;
+  colour: string;
+}
+
+const Icon = (props: IconProps) => {
```

Comment:
Can we use `function` instead of `const` when creating functions please?

Reply:
👍